chart/compass/charts/gateway/templates/oathkeeper-rules.yaml

```diff
@@ -95,6 +95,30 @@ spec:
 ---
 apiVersion: oathkeeper.ory.sh/v1alpha1
 kind: Rule
+metadata:
+  name: compass-broker-certs
+spec:
+  description: Configuration of oathkeeper for secure endpoint of compass system broker
+  upstream:
+    url: "http://compass-system-broker.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.global.system_broker.port }}"
```

Comment:
Do we want to Audit log operations for system broker? If this is the case then may be the requests should go through the compass-gateway component first. Not sure how this will affect authentication/authorization though.

Reply:
We decided to go with another approach. The outgoing requests to the director to go through the compass-gateway component so there are Audit logs (please see https://github.com/kyma-incubator/compass/blob/system-broker-secure-osb-endpoints/chart/compass/charts/system-broker/templates/deployment.yaml#L62-L63)